src/main/java/org/terasology/launcher/packages/GamePackage.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.launcher.packages;
+
+/**
+ * Model of a game package.
+ */
+public class GamePackage {
```

Comment:
I used `GamePackage` class as a placeholder for the actual model of a game package. And yes, I agree with your idea of dropping the 'game' from in front of it, since there are future plans to test it for modules too.